.gitignore

```diff
@@ -1,3 +1,7 @@
 node_modules
 npm-debug.log
 tmp
+package-lock.json
```

Comment:
I've added a `.npmrc` file in #56 as an addition to this which will prevent npm from creating the lock file also